core/build.gradle

```diff
@@ -137,6 +137,12 @@ dependencies {
 
     // Jetty, for testing interoperability with other servers.
     testImplementation 'org.eclipse.jetty:jetty-webapp'
+
+    // Brotli
+    implementation "com.aayushatharva.brotli4j:brotli4j:1.4.2"
```

Comment:
Could we upgrade to the latest version of `brotli4j` which had performance bump?